.github/workflows/ci_e2e_workflow.yml

```diff
@@ -111,7 +120,10 @@ jobs:
           AzureWebJobsEventGridTopicUri: ${{ secrets.LinuxEventGridTopicUriString39 }}
           AzureWebJobsEventGridConnectionKey: ${{ secrets.LinuxEventGridConnectionKeyString39 }}
           ARCHIVE_WEBHOST_LOGS: ${{ github.event.inputs.archive_webhost_logging }}
-        run: .github/Scripts/e2e-tests.sh
+        run: |
```

Comment:
good idea but can we parallelize these scripts execution? ex. expanding them into a submatrix of parallel jobs/steps for speedup. if any fails then e2e fail but also ensure all of scripts complete running so we can capture all errors per run.

Reply:
Discussed and addressed over call.